### PR TITLE
add default and special data sender function for chirpstack integration

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/plugins/integrations/chirpstack.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+
+const downlinkCounters = new Map();
+
+const sendMessage = async ( integration, eui, data ) => {
+    let downlinkCounter = downlinkCounters.get(eui) || 3;
```
Comment (DarkPark) [inline src/plugins/integrations/chirpstack.js]: why 3? maybe move to a constant?

File: src/configs/chirpstack.js [hunk 2]
```diff
@@ -1,6 +0,0 @@
-const {env} = process;
-
-export default {
-    apiUrl: env.CHIRPSTACK_REST_API_URL,
```
Comment (DarkPark) [inline src/configs/chirpstack.js:4]: we really don't need this?

Comment (DeadByElpy) [reply src/configs/chirpstack.js:4]: it was used by 'ns-webhook/chirpstack' route which not exists anymore

File: src/plugins/integrations/chirpstack.js (hunk 1 above)
Comment (DeadByElpy) [reply src/plugins/integrations/chirpstack.js]: left after debugging, set initial value to 1 and add increment to it

**Review by DarkPark — APPROVED** [review]

**Review by Pnom — APPROVED** [review]

**Pull request merged**